file/filename.h

```diff
@@ -46,6 +46,10 @@ enum FileType {
 // operations.
 extern bool ShouldSkipEncryption(const std::string& fname);
 
+// Determine if the content is read from the valid current file.
+extern bool isValidCurrentFile(
+    std::unique_ptr<rocksdb::SequentialFile>* seq_file);
```

Comment:
```suggestion
    std::unique_ptr<rocksdb::SequentialFile> seq_file);
```
This is to indicate the function will consume `seq_file`.